docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Trying to create projections with the same name will result in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#CreateContinuous_Conflict
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+public CompletableFuture createContinuous(final String projectionName, final String query)
+public CompletableFuture createContinuous(final String projectionName, final String query, CreateContinuousProjectionOptions options)
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+createContinuousProjection(projectionName: string, query: string, options?: CreateContinuousProjectionOptions): Promise<void>
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Update a projection
+
+Updates a projection. The name parameter is the name of the projection to be updated. The query parameter contains the new JavaScript.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Update
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+updateProjection(projectionName: string,query: string,options?: UpdateProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Updating a projection that does not exists results in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Update_NotFound
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+updateProjection(projectionName: string,query: string,options?: UpdateProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## List all projections
+
+Returns a list of all projections.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#ListAll
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```
+// This is currently not available in the NodeJS client
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## List continuous projections
```

Comment:
**SUGGESTION:** Would be good to add some context to this operation. Especially what's the Result and partition. Plus, I it might be worth only mentioning listing continuous projections, as listing all will be disabled soon (via RFC).

Reply:
well we have user projection & system projection , if the RFC does not allow that , we have a problem .

Reply:
It seems that [`ListContinuous` should be renamed to `List`](https://github.com/EventStore/home/blob/master/RFCs/021%20-%20gRPC%20Projections.md)
See: 
![obraz](https://user-images.githubusercontent.com/5562528/129678779-c9c2283f-4ea2-40b5-b892-748626f6206b.png)